index.html

```diff
@@ -1,16 +1,72 @@
 <!DOCTYPE html>
 <html>
+    
+    <input type="text" id="mySearch" onkeyup="myFunction()" placeholder="Search.." title="Type in a category">
+
+<ul id="myMenu">
+  <li><a href="file:///Users/osvaldoescobar/Projects/Thinkment/philosophical_issues.html">Philosophical Issues</a></li>
+  <li><a href="file:///Users/osvaldoescobar/Projects/Thinkment/clashes_between_ideas.html">Clashes between Ideas</a></li>
+
+
+    
     <head>
+        <style> 
+        
+#mySearch {
+  width: 20%;
+  font-size: 14px;
+  padding: 9px;
+  border: 4px solid #ddd;
+}
+
+#myMenu {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+#myMenu li a {
+  padding: 10px;
+  text-decoration: none;
+  color: black;
+  display: block
+}
+
+#myMenu li a:hover {
+  background-color: #eee;
+}
+</style>
+<script>
+function myFunction() {
+  // Declare variables
+  var input, filter, ul, li, a, i;
+  input = document.getElementById("mySearch");
+  filter = input.value.toUpperCase();
+  ul = document.getElementById("myMenu");
+  li = ul.getElementsByTagName("li");
+
+  // Loop through all list items, and hide those who don't match the search query
+  for (i = 0; i < li.length; i++) {
+    a = li[i].getElementsByTagName("a")[0];
+    if (a.innerHTML.toUpperCase().indexOf(filter) > -1) {
+      li[i].style.display = "";
+    } else {
+      li[i].style.display = "none";
+    }
+  }
+}
+</script>
```

Comment:
Move [all the code within the script tags](https://github.com/Osvaldo178/Thinkment/pull/6/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R40-R57) into the script.js file you have in [the main folder. ](https://github.com/Osvaldo178/Thinkment/blob/main/script.js)